nat-lab/tests/test_pmtu.py

```diff
@@ -0,0 +1,199 @@
+import config
+import pytest
+from contextlib import AsyncExitStack
+from helpers import SetupParameters, setup_environment, setup_connections
+from telio import AdapterType
+from telio_features import TelioFeatures
+from utils import testing, stun
+from utils.connection_util import (
+    ConnectionTag,
+    generate_connection_tracker_config,
+    ConnectionLimits,
+)
+from utils.ping import Ping
```

Comment:
Hmmm... How do we know our ipv6 implementation works then? are there ways to fix Docker for that or maybe we should reconsider having ipv6 support for MTU. Does it make sense to support it for ipv6 at all?

Reply:
We do have IPv6 VPN server as far as I know so it would be nice to have PMTU support for it. As for docker, it's not really docker's fault but the way local networks of IPv6 are constructed. The problem is that the IP ranges overlap between different network interfaces and kernel doesn't know to which interface bind the socket

Reply:
I would love to learn more about it - is this inherent problem for ipv6 or is this somehow related to our setup?

Reply:
The thing is that this address in present on more than one network https://stackoverflow.com/questions/12260003/connect-returns-invalid-argument-with-ipv6-address